indico/web/client/js/react/forms/final-form.jsx

```diff
@@ -77,6 +77,34 @@ FieldCondition.defaultProps = {
   inverted: false,
 };
 
+/**
+ * A component that sets the value of a form field after the final-form has been initialized.
+ * This can be used when you want to set the initial value of a field in a way that marks the
+ * field as touched and dirty.
+ *
+ * Note that when using this, you most likely need to pass `extraSubscription={{touched: true}}`
+ * to the `FinalSubmitButton` of the form to ensure it correctly gets enabled if the initial value
+ * set using this component is all that's necessary for the submit button to be enabled.
+ */
+export function DirtyInitialValue({field, value}) {
+  const {
+    input: {onChange: setValue, onFocus, onBlur},
+  } = useField(field);
+  useEffect(() => {
+    setTimeout(() => {
+      onFocus();
+      setValue(value);
+      onBlur();
+    });
```

Comment:
That's really dirty, but if that's what it takes to make final form work.. :sweat_smile: